tests/TestCase/Routing/RouterTest.php

```diff
@@ -1897,52 +1916,12 @@ public function testGenerationWithSslOption()
         $result = Router::url([
             '_ssl' => true,
         ]);
-        $this->assertSame('https://localhost/images/index', $result);
-
-        $result = Router::url([
-            '_ssl' => false,
-        ]);
-        $this->assertSame('http://localhost/images/index', $result);
-    }
-
-    /**
-     * Test that the _ssl + _full options work together.
-     *
-     * @return void
-     */
-    public function testGenerationWithSslAndFullOption()
```

Comment:
It might be not clear in the diff but I dropped this method `testGenerationWithSslAndFullOption()`.  Using `_full => false` in combination with `_ssl` made no sense. If you set `_ssl` you will always get a full URL and value of `_full` should be redundant.